File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_06958.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest06958():
    upload_name = request.files['upload'].filename
    data = '{}'.format(upload_name)
    try:
        result = int(str(data))
    except Exception:
        pass
    return jsonify({"result": "success"})

File: Benchmarks/normal/java/spring/testcode/BenchmarkTest06958.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest06958 {

    private static String trimEnds(String v) { return v.trim(); }

    @GetMapping("/BenchmarkTest06958")
    public void BenchmarkTest06958(@RequestHeader("Authorization") String authorization, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String authHeader = authorization != null ? authorization : "";
        String data = trimEnds(authHeader);
        if (!"test".equals(System.getenv("APP_ENV"))) {
            Object evaluated = new org.springframework.expression.spel.standard.SpelExpressionParser().parseExpression(data).getValue();
            response.getWriter().print("<div>" + evaluated + "</div>");
        }
    }
}
